config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigqueryanalyticshubdataexchanges.bigqueryanalyticshub.cnrm.cloud.google.com.yaml

```diff
@@ -8,18 +8,15 @@ metadata:
     cnrm.cloud.google.com/managed-by-kcc: "true"
     cnrm.cloud.google.com/stability-level: alpha
     cnrm.cloud.google.com/system: "true"
-    cnrm.cloud.google.com/tf2crd: "true"
   name: bigqueryanalyticshubdataexchanges.bigqueryanalyticshub.cnrm.cloud.google.com
 spec:
   group: bigqueryanalyticshub.cnrm.cloud.google.com
   names:
     categories:
     - gcp
     kind: BigQueryAnalyticsHubDataExchange
+    listKind: BigQueryAnalyticsHubDataExchangeList
     plural: bigqueryanalyticshubdataexchanges
-    shortNames:
```

Comment:
Nit: Can you add these - you can add annotations to the type definition.

Not blocking, and maybe we should automate generating them